docs/resources/app.md

```diff
@@ -30,7 +30,7 @@ resource "coder_app" "code-server" {
   name     = "VS Code"
   icon     = data.coder_workspace.me.access_url + "/icons/vscode.svg"
   url      = "http://localhost:13337"
-  path     = true
+  relative_path     = true
```

Comment:
@ericpaulsen can you adjust the formatting to align the = signs? `terraform lint` should achieve this

Reply:
`terraform fmt` only applies to `.tf` files, so I manually updated the formatting.